jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh

```diff
@@ -22,9 +22,9 @@ terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.0.0"
 terra_jupyter_r="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.0.0"
 terra_jupyter_bioconductor="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.0.0"
 terra_jupyter_hail="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:1.0.0"
-terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.0"
-terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.6"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.6"
+terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.1"
+terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.1.2"
```

Comment:
I have a couple of questions:

1. Why are we lowering the version of `aou_old`?
2. AoU is using [terra-jupyter-aou:1.1.5](https://github.com/all-of-us/workbench/blob/ba26a89ea09c21e29cec73113e683b44641774de/api/config/config_prod.json#L13). Shouldn't we have been caching that?

Reply:
when I checked, aou was still using 1.1.2...https://terra-bio.slack.com/archives/C9C4KKU65/p1626958772093300

Reply:
AoU is [currently on 1.1.5](https://github.com/all-of-us/workbench/blob/master/api/config/config_prod.json#L13).